client.py

```diff
@@ -0,0 +1,94 @@
+"""
+Implement a tkinter client who communicates with a fastapi server
+and executes commands
+"""
+
+import requests
+from threading import Thread
+import tkinter
+
+
+root = tkinter.Tk()
+frame = tkinter.Frame(root)
+command_inp = tkinter.Text(frame, width=70, height=1, fg="green")
+
+
+def retrieve_input():
+    """
+    Get a command input from the user and get the output to the command
+    from a fastapi server
+    """
+
+    command = command_inp.get("1.0", "end-1c").split(" ")
```

Comment:
Splitting by `" "` will cause errors if the file name or the command has spaces inside of it.